client/src/components/Atoms/Table.tsx

```diff
@@ -51,39 +51,26 @@ export const FilterInput: React.FC<IFilterInputProps> = ({
   </div>
 )
 
-interface IDownloadCSVButtonProps {
-  tableId: string
+export const downloadTableAsCSV = (
+  tableId: string,
   fileName?: string
```

Comment:
nit: Slight preference for an object input here so that 1) consumers can't accidentally mix up the two string fields and 2) the values are labeled in consumer calls for clarity